source/acquaman/AMStepScanActionController.cpp

```diff
@@ -204,6 +204,11 @@ void AMStepScanActionController::onFileWriterError(AMScanActionControllerBasicFi
 		userErrorString = "Your scan has been aborted because Acquaman was unable to open the desired file for writing (for internal storage). This is a serious problem and would have resulted in collecting data but not saving it. Please contact the Acquaman developers immediately.";
 		break;
 
+	case AMScanActionControllerBasicFileWriter::FailedToWriteFile:
+		AMErrorMon::alert(this, AMSTEPSCANACTIONCONTROLLER_FAILE_TO_WRITE_FILE, QString("Error, the %1 Scan Action Controller failed to write your data. This is a serious problem, please contact the Acquaman developers.").arg(stepConfiguration_->technique()));
```

Comment:
@dretrex  is right, this is a cataclysmic problem. We need to escalate it as serious and actually have it cause scan failure.